docs/faq/items/python2_support.md

```diff
@@ -0,0 +1,6 @@
+# Does Black support Python 2?
+
+By default, no. [Install](../../getting_started.md#installation) with the `python2`
```

Comment:
We should start off saying "yes", not "no".

Reply:
Yeah that's probably better 👍